ime/app/src/main/java/com/anysoftkeyboard/ime/AnySoftKeyboardSuggestions.java

```diff
@@ -713,14 +765,27 @@ private void markExpectingSelectionUpdate() {
                 SystemClock.uptimeMillis() + MAX_TIME_TO_EXPECT_SELECTION_UPDATE;
     }
 
+    public void disableSamePunctuation() {
+        mEnableSamePunctuation = false;
+        mIsLastPunctuationSame = 0;
+    }
+
+    public void updateAdditionalCharForReverting(boolean newValue) {
+        mAdditionalCharacterForReverting = newValue;
+    }
+
     protected void handleSeparator(int primaryCode) {
         performUpdateSuggestions();
+
+        Logger.d("nicoursi", "Soooo, this is really a case of handleSeparator()");
```

Comment:
you should probably remove all these debug logs 🙂

Reply:
Yes, i am just waiting for @nicoursi review to remove all of theses ! If he or someby else find a bug, it will be more useful with them !

Reply:
Fair enough 🙂 I'm also installing an apk from this PR now, to help with testing 👍

Reply:
Thanks ! I am using it daily and did not find any bug remaining.